### ISSUE TITLE
Module not found: Error: Can't resolve '../package'

### ISSUE THREAD

**Issue by ramusus** [spencermountain/wtf_wikipedia#239]
after upgrade from 5.3.1 to 7.2.8 I got
```
ERROR in ./node_modules/wtf_wikipedia/src/index.js
Module not found: Error: Can't resolve '../package' in '.../node_modules/wtf_wikipedia/src'
 @ ./node_modules/wtf_wikipedia/src/index.js 4:16-37
 @ ./components/ObjectWikipedia/ObjectWikipedia.jsx
 @ ./components/PersonPage/PersonPage.jsx
 @ ./components/App/App.jsx
 @ ./client/ClientApp.jsx
 @ ./client/index.jsx
 @ multi ./client/index
```

**Comment by spencermountain** [comment]
whoa, thanks. will check this out pronto

**Comment by spencermountain** [comment]
hey @ramusus how are you calling this?
i haven't been able to reproduce this:
```js
const wtf = require('wtf_wikipedia')
console.log(wtf('works').text())
console.log(wtf.version)
//'works'
// 7.2.8
```

**Comment by spencermountain** [comment]
i've also tried it on node 4, in the browser, etc.

**Comment by spencermountain** [comment]
hi, i may have fixed this in 7.2.9, 
wanna check it out? 
cheers
